README.md

```diff
@@ -34,3 +34,19 @@ To see how it works just simply issue at your shell prompt:
 ```
 $ jlc-kicad-tools
 ```
+
+### KiCad
+
+1. Design your board as usual
+1. Create a custom field in Eeschema (for instance LCSCStockCode) and add in the components "C" stock number from JLCPCB/LCSC
+1. Once you are satisfied with your board design, go to Eeschema and run “Tools -> Generate Bill of Materials”
+1. Go to Pcbnew and run ‘File -> Fabrication Outputs -> Footprint Position (.pos) File’.
+1. Plot gerbers as per usual for PCB manufacturing
+1. Run the script and give it the project directory
+
+```
+generate_jlc_files.py --warn-no-lcsc-partnumber --database=cpl_rotations_db.csv --verbose myprojectfolder
+```
+1. Upload your Gerber and CPL/BOM files to JLCPCB
+1. Review the rotations of components on the website, for any that are rotated incorrectly you can override the angle using the "JLCPCBRotation" custom KiCad field, once added, repeat these steps to generate fresh BOM/CPL files (no need to generate Gerber again unless you moved components)
```

Comment:
Please recommend modifying the global database first, so it won't have to be done per-board, and other people can benefit from the corrections. The new field is really only for things like rotating resistors for cosmetic reasons like you said, and maybe if there are components with the same footprint that have different corrections (we still need to figure out how to solve that long term). If something needs to be rotated for correctness, it should be done in the db file.

Reply:
Happy to recommend changing the database first, however I find that not sufficient, the database needs to be more granular even down to the "C" number or part number as I have boards with same sized components but different orientations.

Reply:
I agree. We have had some reports of other examples, too. I would still rather extend the existing database format to allow for per component rotations, though, because there's still fundamentally a correct orientation for the part, not for the instance of the part, and the correction should be applied at the most general level possible for reusability.